domains/devansh.json

```diff
@@ -8,5 +8,14 @@
     },
     "record": {
         "CNAME": "devanshyadav2010.github.io"
+    },
+    "record": {
+        "name": "_discord.devansh.is-a.dev",
+        "TXT": "dh=d14eedf95cd289bfd460f05eff5cca7a8a28d7f0"
+    },
+    "record": {
+        "MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ]
```

Comment:
This is not valid, you can only have one record property in JSON. https://docs.is-a.dev to see what a valid file looks like.

Reply:
Hey sir I'm really unable to understand the docs can you please how can I connect multiple dns.